README.md

```diff
@@ -187,6 +186,22 @@ Flagship.define :production do
 end
 ```
 
+And you can also extend helper methods from base flagset.
+
+```rb
+Flagship.define :base do
+  def staff?(context)
+    def is_author(context)
```

Comment:
I think the example looks a bit complex. Did you really intend to define the method `is_author` inside *another* method `staff?` ?